### DISCUSSION TITLE
Can groops doing PPP with kalman fileter?

### DISCUSSION THREAD

**Discussion by FFFFFG-IGG** [groops-devs/groops#208] [Q&A] [answered]
Hi developers,

Now I'm trying to do PPP with kalman fileter to learn about the convergence process. But as I known, GROOPS is run with a overall matrix and the is ended when all epochs meet the convergence threshold. I wonder if groops can do PPP with kalman filter so that one can know the exact convergence process? Hope to get some suggestions.

regards,
Yongzhao Fan

**Comment by tmayerguerr** [comment, marked as answer]
Dear Yongzhao Fan,

At the moment there is no Kalman filter for PPP implemented in GROOPS. I don't think we will programme something like that in the near future either, as PPP is not our main focus. I am sorry that I cannot give a better answer.

regards
Torsten
